### DISCUSSION TITLE
Double Nested Hierarchy Table of Contents

### DISCUSSION THREAD

**Discussion by tinder-cesardiez** [mkdocs/mkdocs#3274] [Q&A]
Hi there! I'm not sure this is a bug or a feature, but in any case I wanted to ask for feedback.

Currently, MKDocs only supports one level of nested hierarchy when it comes to section titles on the Table of Contents.

### Behavior

So if we have:

```
# Title 1
## Title 2
### Title 3
```

The render output on the ToC is something like:

```
Title 1
    Title 2
    Title 3
```

### Expected

For:

```
# Title 1
## Title 2
### Title 3
```

I would expect the ToC to render:

```
Title 1
    Title 2
        Title 3
```

### Context

- Reproducible on MKDocs 1.4.3
- I understand there is a limit of nested hierarchies, but supporting at least one extra level would be great!

**Comment by oprypin** [comment]
MkDocs itself has no limitation on ToC nesting. You may search for references to `toc_depth` but the default is the full range of `1-6`. However both of the built-in themes in MkDocs and many external themes have a limitation on nesting. This can only be discussed in the context of a specific theme.
